python/langsmith/client.py

```diff
@@ -4811,3 +5364,72 @@ def _tracing_sub_thread_func(
         tracing_queue, limit=size_limit, block=False
     ):
         _tracing_thread_handle_batch(client, tracing_queue, next_batch)
+
+
+def convert_prompt_to_openai_format(
+    messages: Any, stop: Optional[List[str]] = None, **kwargs: Any
+) -> dict:
+    """Convert a prompt to OpenAI format.
+
+    Requires the `langchain_openai` package to be installed.
+
+    Args:
+        messages (Any): The messages to convert.
+        stop (Optional[List[str]]): Stop sequences for the prompt.
+        **kwargs: Additional arguments for the conversion.
+
+    Returns:
+        dict: The prompt in OpenAI format.
+    """
+    try:
+        from langchain_openai import ChatOpenAI
+    except ImportError:
+        raise ImportError(
+            "The convert_prompt_to_openai_format function requires the langchain_openai"
+            "package to run.\nInstall with `pip install langchain_openai`"
+        )
+
+    openai = ChatOpenAI()
+
+    try:
+        return openai._get_request_payload(messages, stop=stop, **kwargs)
+    except Exception as e:
+        raise ls_utils.LangSmithError(f"Error converting to OpenAI format: {e}")
+
+
+def convert_prompt_to_anthropic_format(
+    messages: Any,
+    model_name: str = "claude-2",
```

Comment:
Why default to cluade-2? Feels like it's gonna be deprecated soon.

Also why make only `stop` configurable and not the base url, etc.? Would probably be better tojust ahve a `model_kwargs` or something

Reply:
changed it to model_kwargs and defaulted to "claude-3-haiku-20240307"